pkg/virt-handler/node-labeller/cpu_plugin_test.go

```diff
@@ -206,6 +206,39 @@ var _ = Describe("Node-labeller config", func() {
 		}
 		Expect(badKey).To(BeEmpty())
 	})
+
+	It("Skip node labels should work in runtime.", func() {
```

Comment:
I feel this test could either be renamed to something a bit more clear. When I look at this description, I interpret the meaning as the same test case as the following test about the LabellerSkipNodeAnnotation, while both tests have the exact opposite expected results. 

Since both of these tests are testing the functionality of the skipNode function at runtime, I would recommend naming them something similar to reflect that they are testing the same function, just the different expected results from it.

Reply:
fixed